architecture/core_concepts/routes.adoc

```diff
@@ -322,9 +322,19 @@ In order for services to be exposed externally, an {product-title} route allows
 you to associate a service with an externally-reachable host name. This edge
 host name is then used to route traffic to the service.
 
-When two routes claim the same host, the oldest route wins. If additional routes
-with different path fields are defined in the same namespace, those paths will be
-added. If multiple routes with the same path are used, the oldest takes priority.
+When multiple routes from different namespaces claim the same host,
+the oldest route wins and claims it for the namespace. If additional
+routes with different path fields are defined in the same namespace,
+those paths will be added. If multiple routes with the same path are
+used, the oldest takes priority.
+
+Please note that a consequence of this behavior is that if you have
+two routes for a hostname: an older one and a newer one.  And if
+someone else has a route for the same hostname that they created
```

Comment:
Maybe `And if another route uses/claims/asks for the same hostname` since it could be someone else or another route (created by the same user).